CryptoPkg/Library/BaseCryptLibMbedTls/BaseCryptLib.inf

```diff
@@ -58,9 +60,10 @@
 
   SysCall/CrtWrapper.c
   SysCall/DummyOpensslSupport.c
-  SysCall/BaseMemAllocation.c
   SysCall/TimerWrapper.c
 
+  $(OPENSSL_PATH)/crypto/sm3/sm3.c

```

Comment:
Could add sm3 file in MbedtlsLib? Then you don't have to add it to every CryptLib.inf

Reply:
Done